### ISSUE TITLE
Auto Jupyter Hub for PRs?

### ISSUE THREAD

**Issue by anderspitman** [biocore/scikit-bio#1202]
Just had an idea. What do you guys think about making a tool for PRs where once a PR passes all checks (CI, ASV, Coverage, etc), a Jupyter Hub is automatically hosted using the branch from the PR. This way multiple people can bang on it and get a feel for functionality/find bugs. Might be useful for code reviews. This could run on powertrip.

As a user story, I just submitted #1201. It would be nice if you guys could quickly drop into a notebook and see what the reprs look like yourselves, rather than reading through the code or pulling my branch and running it.

**Comment by wasade** [comment]
That would be awesome. Preload cells from the cookbook or something?

**Comment by anderspitman** [comment]
Yeah I like that idea. The purpose would probably mostly for UX and visual testing. Things we can't really confirm as easily with automated tests. So loading from the cookbook would give a great starting point and also provide a set of smoke tests. Also has the added benefit of battle testing the cookbook regularly.

**Comment by jairideout** [comment]
:+1: [binder](http://mybinder.org/) may be useful for this.
